code/Template.php

```diff
@@ -39,19 +39,23 @@ public function partial( $template, $params = array() ) {
 	 * @return string|bool
 	 * @throws Exception
 	 */
-	public function render() {
+	public function render(bool $showAdvanced = false) {
 		$file_name = BP . DS . $this->_template;
-
 		// Check for template file.
 		if ( empty( $this->_template ) or ! file_exists( $file_name ) or ! is_readable( $file_name ) ) {
 			throw new Exception( 'Could not find template "' . $file_name . '"! Please check for file existence.' );
 			return false;
 		}
-
+		
 		// Output template.
 		ob_start();
 		include( $file_name );
-		$output = ob_get_contents();
+		// Show advanced form when requested
+		if( isset($this->_vars['form']) && ! empty($this->_vars['form'])  && $showAdvanced){
```

Comment:
This is not the right place for rendering advanced form. `Template` class is meant to be generic template renderer.

Implement advanced form rendering as it was before commit https://github.com/sendsmaily/sendsmaily-wordpress-plugin/commit/6802f90104e7cd3109bbfdfe10a5a5816b42943a (advanced form existence was checked in the template).